### ISSUE TITLE
Fails to large long export

### ISSUE THREAD

**Issue by talios** [revapi/osgi-revapi-extension#1]
Using the latest HEAD build of the extension ( which thankfully added logging for this failure ) I see that I'm getting:

```
java.lang.IllegalArgumentException: Could not parse the Export-Package directive. Errored on index 205 of directive:
com.smxemail.api;version="22.0.1";uses:="com.fasterxml.jackson.annotation,com.fasterxml.jackson.databind.annotation,com.smxemail.datatype,freemarker.template,javax.naming.ldap,org.springframework.ldap.core"
```

I started adding this as a test to `ExportPackageEntryParserTest` and indeed the test fails. Interestingly - if I drop a single character from the string - so that it ends with `ldap.cor` - the test passes.

**Comment by metlos** [comment]
That is rather weird..

I'm heading offline for the Easter holiday, so unfortunately I won't be able to look at it sooner than sometime late next week, but maybe in the meantime you'll find out what's wrong.

Thanks for reporting it!

**Comment by talios** [comment]
@metlos Soon to head off work work lunch ( my 10 year celebration here ) along with Easter as well, will see if I can figure out whats going on.

**Comment by talios** [comment]
@metlos did you ever get a chance to look into this? I never got a change to look closely myself, and couldn't quite figure out what the parser was doing when I did.

**Comment by metlos** [comment]
Not really.. Was hoping you were on it...

**Comment by talios** [comment]
@metlos Looks like I've solved the above issues in the current PR I submitted.

**Comment by talios** [comment]
@metlos any chance of taking a look at that PR, and hopefully merging/releasing?

**Comment by metlos** [comment]
should be fixed in `osgi-revapi-extension-0.1.1`. Thanks for all the work on this!
